android/build.gradle

```diff
@@ -0,0 +1,43 @@
+buildscript {
+  repositories {
+    google()
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.6.0'
+  }
+}
+
+apply plugin: 'com.android.library'
+
+def safeExtGet(prop, fallback) {
+  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+def DEFAULT_COMPILE_SDK_VERSION = 28
+def DEFAULT_BUILD_TOOLS_VERSION = '28.0.2'
+def DEFAULT_MIN_SDK_VERSION = 19
+def DEFAULT_TARGET_SDK_VERSION = 27
+
+android {
```

Comment:
Please add the package name in here: ` namespace "app.edge.reactnative.currencyplugins"`, and then remove it from the Android Manifest.

Putting the namespace in the manifest works, but is deprecated.

Reply:
Hmm, this is not how it is configured in accountbased.

Reply:
See edge-core-js for an example. I guess we need to update accountbased too.